src/anshuKJun20/SwapNumber.java

```diff
@@ -0,0 +1,22 @@
+package anshuKJun20;
+
+public class SwapNumber {
+
+	private void swapMethod(int number1, int number2) {
+		System.out.println("Input = ("+number1+" "+number2+")");
+		 number1 = number1*number2;
+		 number2 = number1/number2;
+	     number1 = number1/number2;
```

Comment:
Anything inside block should be indented.For proper indentation use ctrl+shift+f

Reply:
done